source/agora/node/FullNode.d

```diff
@@ -269,6 +285,7 @@ public class FullNode : API
     /// GET: /node_info
     public override NodeInfo getNodeInfo () pure nothrow @safe
     {
+        endpoint_request_stats.increaseAgoraEndpointCallsTotalBy(1, "getNodeInfo", "http");
```

Comment:
I guess this is the only call so far just for testing ?

Reply:
~Also where is `increaseAgoraEndpointCallsTotalBy` defined ?~ Ah found it, it's generated
It seems weird to have a "Agora endpoints call total", instead we just give the most precise info (GET to endpoint XXX) and let upstream generate aggregations.

Reply:
a lot of Prometheus function is dependent on monotonically increasing metrics like 
rate(prometheus_tsdb_head_chunks_created_total[1m])

I think when it comes to http_calls, the Promethues examples also espect aggregated statistics for example:
prometheus_http_requests_total{code="200",handler="/api/v1/label/:name/values"} 1

Reply:
> I guess this is the only call so far just for testing ?

yes for now, I only added one call, I am planning to do the rest in a similar manner

Reply:
> a lot of Prometheus function is dependent on monotonically increasing metrics like
> rate(prometheus_tsdb_head_chunks_created_total[1m])
> 
> I think when it comes to http_calls, the Promethues examples also espect aggregated statistics for example:
> prometheus_http_requests_total{code="200",handler="/api/v1/label/:name/values"} 1

also Promethues has some built-in timeseries 'fix' for the total count: if we restart agora node and the total count starts from 0, Promethues will detect it

Reply:
Thanks for the explanation. It LGTM.